tests/unit/specs/components/AboutMe/AboutMyPersonalInfo/AboutMyPersonalInfo.nuxt.spec.ts

```
@@ -0,0 +1,64 @@
+
+  describe("Phone Number", () => {
+    it("should render formatted phone number when rendered.", () => {
+      const phoneNumber = wrapper.find<HTMLDivElement>("#phone-number");
+
+      expect(phoneNumber.text()).toBe("12 34 56 78 90");
+    });
+
+    it("should render ? when phone number in runtime config doesn't have a correct format.", async() => {
+      wrapper = await mountAboutMyPersonalInfoComponent();
+      const phoneNumber = wrapper.find<HTMLDivElement>("#phone-number");
+      mockNuxtImport("useRuntimeConfig", () => () => ({
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)